init.fish

```diff
@@ -3,7 +3,14 @@ function init -a path --on-event init_nvm
 
   if available fenv
     set -q NVM_DIR; or set -gx NVM_DIR ~/.nvm
-    fenv source $NVM_DIR/nvm.sh >/dev/null ^&1
+
+    if available brew; and brew --prefix nvm > /dev/null
```

Comment:
you are right, I tried the negative side in a wrong way (brew --prefix something_doesnt_in_homebrew).